Makefile.am

```diff
@@ -1,22 +1,7 @@
-#
 # Copyright 2008-2020 Red Hat, Inc.
-#
-# This program is free software; you can redistribute it and/or modify
-# it under the terms of the GNU General Public License as published by
-# the Free Software Foundation; either version 2 of the License, or
-# (at your option) any later version.
-#
-# This program is distributed in the hope that it will be useful,
-# but WITHOUT ANY WARRANTY; without even the implied warranty of
-# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-# GNU General Public License for more details.
-#
-# You should have received a copy of the GNU General Public License
-# along with this program; if not, write to the Free Software
-# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
-#
-# Refer to the README and COPYING files for full details of the license
-#
+# SPDX-FileCopyrightText: 2022 Red Hat, Inc.
```

Comment:
You can use "reuse addheader --exclude-year" to avoid the year.

It will also help to document or add a script with the complete reuse command line needed to add a new file or to fix existing files. In this script you can document
why you use the specific options.

Reply:
Also discussed last week (https://github.com/oVirt/vdsm/pull/306#issuecomment-1235441849). I saw the year can be omitted in the command line, but are we sure we can just discard them (legally speaking)?

Reply:
I don't see any issue, this is why reuse provides --exclude-year